frontend/src/components/RegisterEmployer/index.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+
+import SRegisterEmployer from "./style";
+
+export default function RegisterEmployer() {
+  const [lastname, setLastname] = useState("");
+  const [firstname, setFirstname] = useState("");
+  const [email, setEmail] = useState("");
+  const [selection, setSelection] = useState("");
+
+  const hChangeLastname = (evt) => {
```

Comment:
Possibility to make one function instead of 3

Reply:
Do what he says